### PR TITLE
Force Chrome to clear cache before opening a URL

### PR DESCRIPTION
**Pull request by kyriakos**

### CODE REVIEW

**Comment by westy92** [comment]
Please make this an option, and not something that happens every time.

Add the property `clearCache` (boolean) to `CreateOptions`.

File: src/index.ts [hunk 1]
```diff
@@ -61,7 +61,9 @@ async function generate(html: string, options: CreateOptions): Promise<CreateRes
   await throwIfCanceled(options);
   const client = await CDP(options);
   try {
+    const {Network} = client;
     const {Page} = client;
```
Comment (westy92) [inline src/index.ts]: Combine into:
```typescript
const {Network, Page} = client;
```

File: src/index.ts [hunk 2]
```diff
@@ -61,7 +61,9 @@ async function generate(html: string, options: CreateOptions): Promise<CreateRes
...
+    await Network.clearBrowserCache(); // Clear cache to force chrome to load a fresh copy
```
Comment (westy92) [inline src/index.ts]: Check for the added option (see PR comment):

```typescript
if (options.clearCache) {
  await Network.clearBrowserCache();
}
```

Comment (westy92) [reply src/index.ts]: Throw a `await throwIfCanceled(options);` above this `if` check, too.

Comment (kyriakos) [reply src/index.ts]: good idea. will do and update.

**Comment by westy92** [comment]
Fixed in 652502fd0d400696a25f0c05c6c9930f9c782f0b.

**Comment by westy92** [comment]
Usage:

```typescript
const options: HtmlPdf.CreateOptions = {
  port,
  clearCache: true,
};
```

**Pull request closed without merging**